### ISSUE TITLE
еще языки

### ISSUE THREAD

**Issue by solanum-tuberosum** [obelyaev/abaevdict#191]
![image](https://user-images.githubusercontent.com/13102259/97439028-b45f3980-1936-11eb-8067-a0eb3f890e22.png)
тунгус. - записала в прекоммент просто
но есть еще майт.?

(слово færæt)

**Comment by jvsntsn** [comment]
![image](https://user-images.githubusercontent.com/48504970/97714845-b5c56900-1ad2-11eb-8062-7402c6367e41.png)
еще вот этот непонятный товарищ

**Comment by jvsntsn** [comment]
еще новоперсидский бы?
![image](https://user-images.githubusercontent.com/48504970/97720541-72222d80-1ad9-11eb-9811-43e12a5bcf0f.png)
слово abyraeg

или сойдет просто перс?

**Comment by jvsntsn** [comment]
![image](https://user-images.githubusercontent.com/48504970/97725600-6afe1e00-1adf-11eb-854e-29f9e48d6c2f.png)
еще др.болг))))))
слово acawun

**Comment by obelyaev** [comment]
> ![image](https://user-images.githubusercontent.com/13102259/97439028-b45f3980-1936-11eb-8067-a0eb3f890e22.png)
> тунгус. - записала в прекоммент просто
> но есть еще майт.?
> 
> (слово færæt)

Да, тунгусский правильно, конечно, это же просто другое название для эвенкийского (старое). А вот "майт" не знаю, что за язык. Спрошу у коллег, может, знают.

**Comment by obelyaev** [comment]
> еще новоперсидский бы?
> ![image](https://user-images.githubusercontent.com/48504970/97720541-72222d80-1ad9-11eb-9811-43e12a5bcf0f.png)
> слово abyraeg
> 
> или сойдет просто перс?

Вообще Абаев везде пишет "перс.", а древнеперсидский называет "др.перс.", среднеперсидский — "пехл.". Так что скорее всего можно просто "перс." Вам ещё где-то встретилось?

**Comment by obelyaev** [comment]
Выяснил, что майт=кайт (кайтагский), но Юля уже зашла, когда я разлочил :)) Если "кайт." есть, то его и впишите туда.

**Comment by jvsntsn** [comment]
> Выяснил, что майт=кайт (кайтагский), но Юля уже зашла, когда я разлочил :)) Если "кайт." есть, то его и впишите туда.

кайт. есть; про новоперсидский не помню сходу, к сожалению. Я вышла из базы, если надо снова залочить ее сейчас)

**Comment by solanum-tuberosum** [comment]
семнани
![image](https://user-images.githubusercontent.com/13102259/98101038-bb3ffc00-1ea2-11eb-8a20-f84c2dd60cd4.png)

**Comment by jvsntsn** [comment]
![image](https://user-images.githubusercontent.com/48504970/98483231-a8695680-2217-11eb-818f-6e733568b510.png)
слово ard

**Comment by solanum-tuberosum** [comment]
Слово araq
![image](https://user-images.githubusercontent.com/13102259/98635924-dd1fff80-2336-11eb-8c51-7fd24cf900f2.png)

**Comment by solanum-tuberosum** [comment]
Слово Вадима (@hyppocentaurus ) aslam
![image](https://user-images.githubusercontent.com/13102259/98640740-9da7e200-233b-11eb-909a-c37595a5497f.png)
по идее в скобках языки, а не диалекты. но при этом из них есть только казан. Что делать с этим словом?

**Comment by jvsntsn** [comment]
Я бы записала все языки в цит и сделала бы HideL, а то что у Абаева записала просто текстом...

**Comment by solanum-tuberosum** [comment]
Слово art, не нашла кушан.
![image](https://user-images.githubusercontent.com/13102259/98664478-7b25c100-235b-11eb-8a9b-22f1e0cfac96.png)
